app/models/channel.rb

```diff
@@ -57,32 +57,19 @@ class Channel < ApplicationRecord
     in: %w[domain_not_found connection_failed too_many_redirects timeout no_txt_records token_incorrect_dns token_not_found_dns token_not_found_public_file no_https]
   }, allow_nil: true
 
-  validate :site_channel_details_brave_publisher_id_unique_for_publisher, if: -> {
-                                                                                T.bind(self, Channel)
-                                                                                details_type == "SiteChannelDetails"
-                                                                              }
-
-  validate :verified_duplicate_channels_must_be_contested, if: -> { T.bind(self, Channel).verified? }
-
-  after_save :notify_slack, if: -> {
-                                  T.bind(self, Channel)
-                                  :saved_change_to_verified? && verified?
-                                }
-  before_save :set_derived_brave_publisher_id, if: -> {
-                                                     T.bind(self, Channel)
-                                                     derived_brave_publisher_id.nil?
-                                                   }
+  validate :site_channel_details_brave_publisher_id_unique_for_publisher, if: -> { details_type == "SiteChannelDetails" }
+
+  validate :verified_duplicate_channels_must_be_contested, if: -> { verified? }
+
+  after_save :notify_slack, if: -> { saved_change_to_verified? && verified? }
+
+  before_save :set_derived_brave_publisher_id, if: -> { derived_brave_publisher_id.nil? }
 
   # *ChannelDetails get autosaved from above.
-  after_save :update_site_banner_lookup!, if: -> {
-                                                T.bind(self, Channel)
-                                                :saved_change_to_verified? && verified?
```

Comment:
The `:saved_change_to_verified?` will always return true since it's a symbol. But the method is never called, so this will always update the site banner, even if no change to verification has happened.